tests/unit/resources/test_attachment.py

```diff
@@ -24,26 +24,26 @@ def attachments_tester(attachment):
 
 
 @responses.activate
-def test_should_get_attachment(ENDPOINT_BASE, attachment):
+def test_should_get_attachment(client, ENDPOINT_BASE, attachment):
     responses.add(
         responses.GET,
         ENDPOINT_BASE + f"/experiments/{EXP_ID}/attachments",
         json=[attachment.to_dict()],
     )
-    att = Attachment.get(EXP_ID, attachment._id)
+    att = client.get_attachment(EXP_ID, attachment._id)
     attachments_tester(att)
 
 
 @responses.activate
-def test_should_create_attachment(ENDPOINT_BASE, experiment, attachments):
+def test_should_create_attachment(client, ENDPOINT_BASE, experiment, attachments):
     """Test creation of a new attachment.
     This test must be run from the project root directory"""
     responses.add(
         responses.POST,
         ENDPOINT_BASE + f"/experiments/{EXP_ID}/attachments",
         json=attachments[0],
     )
-    att = Attachment.upload(experiment._id, "tests/data/text.txt")
+    att = client.upload_attachment(experiment._id, "tests/data/text.txt")
```

Comment:
With last two comments: These are breaking changes. Can we keep the class methods?

Reply:
Yes, of course; I'll add them back in.

Reply:
added back in; pending for other entities